hi-ml-azure/src/health_azure/himl.py

```diff
@@ -295,7 +326,10 @@ def submit_to_azure_if_needed(  # type: ignore
         submit_to_azureml: Optional[bool] = None,
         tags: Optional[Dict[str, str]] = None,
         after_submission: Optional[Callable[[Run], None]] = None,
-        hyperdrive_config: Optional[HyperDriveConfig] = None) -> AzureRunInfo:  # pragma: no cover
+        hyperdrive_config: Optional[HyperDriveConfig] = None,
+        num_cross_validation_splits: int = 1,
+        cross_validation_metric_name: str = "val/loss"
```

Comment:
I'm not sure we should bloat the interface of submit_if_needed even more. in addition, we need one more argument that is the name of the commandline arg. It would be just as good if we suggest to users to pass in the result of your helper function in hyperdrive_config.